composer/swipe-composer.js

```diff
@@ -193,7 +199,7 @@ exports.SwipeComposer = Composer.specialize( /** @lends SwipeComposer# */ {
                 return;
             }
 
-            document.removeEventListener("touchmove", this, true);
+            this._removeEventListeners();
```

Comment:
I'm not sure If it's a big deal or not, but yeah in order to keep the same logic in our framework (we chosen that way for the translate composer for exemple)